webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/__tests__/TypeAheadSelectSelectors.test.js

```diff
@@ -0,0 +1,49 @@
+import { selectSelected, selectOptions } from '../TypeAheadSelectSelectors';
```

Comment:
If you use `testSelectorsSnapshotWithFixtures` it will help make this file much cleaner!